src/veh_interact.cpp

```diff
@@ -1396,23 +1412,25 @@ void veh_interact::display_contents()
                 }
             }
 
-        // @todo display contents of vehicle tanks and batteries
+        } else if( pt->is_battery() ) {
+            hdr += string_format( " (%i/%i)", pt->ammo_remaining(), pt->ammo_capacity() );
 
-        } else {
-            continue;
+        } else if( pt->is_tank() ) {
+            if( pt->ammo_remaining() > 0 ) {
+                msg = string_format( "%s (%i)",
+                                     item::nname( pt->ammo_current(), pt->ammo_remaining() ).c_str(),
+                                     pt->ammo_remaining() );
```

Comment:
If it lists exact ammo count rather than just percentage, it could also list maximum ammo count.